terraform/modules/service/roles.tf

```diff
@@ -0,0 +1,24 @@
+# TODO: This may not be the best way to split files, consider in CLDC-2526
```

Comment:
I've added an AC to this ticket to check for TODOs, as we're not in the habit of checking for these yet 👍